src/main/java/com/amazonaws/kinesisvideo/demoapp/activity/WebRtcActivity.java

```diff
@@ -194,6 +200,10 @@ public void onSdpAnswer(final Event answerEvent) {
                 final SessionDescription sdpAnswer = new SessionDescription(SessionDescription.Type.ANSWER, sdp);
 
                 localPeer.setRemoteDescription(new KinesisVideoSdpObserver(), sdpAnswer);
+                Log.d(TAG, "Answer Client ID: " + answerEvent.getSenderClientId());
+                peerConnectionFoundMap.put(answerEvent.getSenderClientId(), localPeer);
```

Comment:
I assume this will replace an existing entry (if one exists).

Reply:
Yes. That is right.